plot_widget.py

```diff
@@ -14,42 +16,67 @@
     QVBoxLayout,
 )
 
+LOG_FILES_DIR = 'logs'
+if not os.path.isdir(LOG_FILES_DIR):
+    os.makedirs(LOG_FILES_DIR)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+formatter = logging.Formatter('%(asctime)s.%(msecs)03d %(levelname)s:%(name)s:%(message)s')
+file_handler = logging.FileHandler('logs/plot_widget.log', mode='w')
+file_handler.setFormatter(formatter)
+logger.addHandler(file_handler)
+
```

Comment:
did we want one for each file?
we should probably talk to Ashley about how much logging he wants